pyvrp/Population.py

```diff
@@ -88,11 +89,17 @@ def add(self, individual: Individual):
         ----------
         individual
             Individual to add to the population.
+        penalty_manager
+            PenaltyManager to use to compute the cost. Required here
+            since adding an individual may trigger a purge which needs to
+            compute the biased fitness which requires computing the cost.
```

Comment:
Part of this is an implementation detail that does not have to be explained here, I think. I would move the part about "Required here because ..." into a comment.

Reply:
Done